package.json

```diff
@@ -59,12 +59,15 @@
     ]
   },
   "dependencies": {
+    "@types/bunyan": "^1.8.0",
```

Comment:
I will remove the complete example and make al link to an own repo only site.  I thought it was a good idea to make also examples in different languages, but as you mentioned with eventprovider.js , examples can also be replace with pseudo code, or links to complete independent repos.